test/parallel/test-process-argv-0.js

```diff
@@ -14,15 +9,10 @@ if (process.argv[2] !== 'child') {
   });
 
   var childArgv0 = '';
-  var childErr = '';
   child.stdout.on('data', function(chunk) {
     childArgv0 += chunk;
   });
-  child.stderr.on('data', function(chunk) {
-    childErr += chunk;
-  });
   child.on('exit', function() {
```

Comment:
Actually, `common.mustCall` makes sure that the `exit` handler is called for sure. Because the value is asserted only inside it. If the handler is never called, the test will still pass.